lib/api.js

```diff
@@ -28,20 +28,15 @@ const { AssetComputeWorkerPipeline } = require('./worker-pipeline');
  * @param {*} options optional options
  */
 function worker(renditionCallback, options = {}) {
-    console.log("## worker(renditionCallback, options={})");
+    console.log("worker(renditionCallback, options={})");
 
     if (typeof renditionCallback !== "function") {
         throw new Error("renditionCallback must be a function");
     }
 
     return actionWrapper(async function (params) {
     
-        // Debug: Use sleep to ensure tests have time to grab logs
-        // const { promisify } = require('util');
-        // const sleep = promisify(setTimeout);
-        // await sleep(2000);
-        console.log("## worker(renditionCallback) -> actionWrapper");
-        //*/
+        console.log("worker(renditionCallback) -> actionWrapper");
```

Comment:
I think we can remove this if we keep the log from L49.

Reply:
+1